.github/workflows/ci.yml

```diff
@@ -38,4 +42,8 @@ jobs:
           coverage run -m unittest discover -s src
           coverage xml
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v2
\ No newline at end of file
+        uses: codecov/codecov-action@v2
+        with:
+          flags: python27
+          name: python-27
+          fail_ci_if_error: true
```

Comment:
Not having new line at the end of the file getting flagged. Please include here and anywhere else.

Part of the reason for this is to avoid an unchanged line at the end of the file looking like it changed when more lines are added - incidentally like in this PR on the old line 41.